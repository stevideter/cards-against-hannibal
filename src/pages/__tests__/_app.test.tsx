import { NextPage } from 'next';
import { AppProps } from 'next/app';
import renderer from 'react-test-renderer';
import MyApp from '../_app';

describe('MyApp', () => {
    const TestPage: NextPage = () => {
        return <div />;
    };
    it('renders correctly', () => {
        const appProps: AppProps = {
            pageProps: {},
            Component: TestPage,
            router: {} as any,
        };
        const tree = renderer.create(<MyApp {...appProps} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
