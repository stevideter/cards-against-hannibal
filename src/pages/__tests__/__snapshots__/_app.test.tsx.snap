// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyApp renders correctly 1`] = `<div />`;
